.circleci/config.yml

```diff
@@ -213,7 +221,7 @@ workflows:
             - test-e2e-firefox
             - test-e2e-chrome-flask
             - test-e2e-firefox-flask
-            - test-e2e-chrome-vault-decryption
+            - test-e2e-chrome-dist
```

Comment:
do we also want the `test-e2e-firefox-dist` in the requires?

Reply:
Thanks, good catch, fixed in ea597fe9db